src/components/NavBar.js

```diff
@@ -1,17 +1,23 @@
 import { Link } from 'react-router-dom';
+import { FaUser } from 'react-icons/fa';
 
 const NavBar = () => (
   <header>
-    <h1 className="logo">Bookstore CMS</h1>
     <nav className="header__nav">
-      <ul>
-        <li>
-          <Link to="/">Books</Link>
-        </li>
-        <li>
-          <Link to="categories">Categories</Link>
-        </li>
-      </ul>
+      <div className="navbar">
+        <h1 className="logo">Bookstore CMS</h1>
+        <ul>
+          <li className="navBooks">
+            <Link to="/">BOOKS</Link>
+          </li>
+          <li className="navCat">
+            <Link to="categories">CATEGORIES</Link>
+          </li>
+        </ul>
+      </div>
+      <div className="user">
+        <FaUser />
+      </div>
```

Comment:
- [ ] Kindly re-work the Navbar so the logo(Bookstore CMS) displays nicely and is not wrapped and also the icon spans to the right. See the screenshots below.

**The original design**
![image](https://user-images.githubusercontent.com/50941074/191777485-378f6362-5176-40af-990a-79a3059053da.png)

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/191777322-b0829f9f-3103-42d6-aba6-2bb311fd4c4d.png)